From 85bc798fab86f2ced349114e1707661154bb8eb9 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 20 Jul 2021 16:20:15 +0200
Subject: IWYU: missing include for std::vector usage in devtools embedded
 message dispatcher.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix GCC build breakage:
    ../../chrome/browser/devtools/devtools_embedder_message_dispatcher.h:116:36: error: ‘vector’ in namespace ‘std’ does not name a template type
      116 |                         const std::vector<base::Value>& params) = 0;
          |                                    ^~~~~~

Bug: 819294
Change-Id: I379bf14416c3bfd3c0cdac7bb2542b79781261ec
---
 chrome/browser/devtools/devtools_embedder_message_dispatcher.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/devtools/devtools_embedder_message_dispatcher.h b/chrome/browser/devtools/devtools_embedder_message_dispatcher.h
index 12f8500b7811..4007112f3941 100644
--- a/chrome/browser/devtools/devtools_embedder_message_dispatcher.h
+++ b/chrome/browser/devtools/devtools_embedder_message_dispatcher.h
@@ -8,6 +8,7 @@
 #include <map>
 #include <memory>
 #include <string>
+#include <vector>
 
 #include "base/callback.h"
 #include "ui/gfx/geometry/insets.h"
-- 
2.30.2

